test/integration/targets/nxos_config/tests/common/save.yaml

```diff
@@ -12,12 +12,11 @@
     parents:
       - "interface {{ intname }}"
     match: none
-    provider: "{{ connection }}"
 
 - name: save config
   nxos_config:
     save_when: always
-    provider: "{{ connection }}"
+    timeout: 300
```

Comment:
The top-level provider spec has been removed. This is no longer a valid attribute. CI failure:
https://object-storage-ca-ymq-1.vexxhost.net/v1/a0b4156a37f9453eb4ec7db5422272df/ansible_68/65068/f7c47b0036d8aac0f5ca98154b87e1582bb903fd/third-party-check/ansible-test-network-integration-nxos-cli-python36/253142f/controller/ara-report/result/7ab707d4-5474-402e-9c99-e3c5a6b78627/